website/src/app/views/our-team/members.js

```diff
@@ -130,6 +130,12 @@ const members = {
         linkedinURL: 'https://www.linkedin.com/in/justin-liao-1949b3217/',
         imageURL: justin,
       },
+      {
```

Comment:
The directions might've been a little ambiguous but the alphabetical order comes after all of your leads' profiles - so go ahead and swap Zoya back and we should be good here